Explorer/Assets/DCL/Profiles/Repository/RealmProfileRepository.cs

```diff
@@ -98,11 +98,11 @@ private static GetProfileJsonRootDto NewProfileJsonRootDto(Profile profile, stri
                 URLAddress url = Url(id, version);
                 var response = webRequestController.GetAsync(new CommonArguments(url), ct, ignoreErrorCodes: IWebRequestController.IGNORE_NOT_FOUND);
 
-                using GetProfileJsonRootDto root = await response.CreateFromNewtonsoftJsonAsync<GetProfileJsonRootDto>(
+                using GetProfileJsonRootDto? root = await response.CreateFromNewtonsoftJsonAsync<GetProfileJsonRootDto>(
                     createCustomExceptionOnFailure: (exception, text) => new ProfileParseException(id, version, text, exception),
                     serializerSettings: SERIALIZER_SETTINGS);
 
-                var profileDto = root.FirstProfileDto();
+                var profileDto = root?.FirstProfileDto();
```

Comment:
WDYT if we add a null check and a return for null above here?

Reply:
The check for nullity is right below this operation, seems kind of an overhead to me